app/frontend/src/components/admin/AdminPage.vue

```diff
@@ -27,8 +28,17 @@ export default {
   },
   data() {
     return {
+      tab: null,
       adminDashboardUrl: this.$config.adminDashboardUrl,
     };
   },
+  computed: {
+    ...mapGetters('form', ['isRTL']),
+  },
+  watch: {
+    isRTL() {
+      this.tab = null;
```

Comment:
This does have the effect of changing which tab is selected when switching between LTR and RTL languages. Since it's the admin page and it's a pretty weird use case, shall we leave it for now and fix later?

Reply:
I could not find any alternative solution.

Reply:
Why is it necessary to set the tab to null when changing to a RtoL language?